.aws/src/event-rules/prospect-events/prospectEventRules.ts

```diff
@@ -94,10 +154,83 @@ export class ProspectEvents extends Resource {
     new PocketEventBridgeRuleWithMultipleTargets(
       this,
       `${config.prefix}-Prospect-Prod-EventBridge-Rule`,
+      prospectGenerationEventRuleProps
+    );
+  }
+
+  /**
+   * Create an event bridge rule for Prospect events.
+   *
+   * @param name
+   * @param source
+   * @param detailType
+   */
+  private createProspectEventRule(
+    name: string,
+    source: string,
+    detailType: string[]
+  ) {
+    const targets: PocketEventBridgeTargets[] = [
+      {
+        arn: this.snsTopic.arn,
+        deadLetterArn: this.sqsDlq.arn, //using the same DLQ for all prospect events (prospect-generation and prospect-dismiss as of now)
+        targetId: `${config.prefix}-Prospect-Event-SNS-Target`,
```

Comment:
@sri-kirsh I forgot but do we need a unique `targetId` here for each Prospect event that will use the SNS or can we keep it the same for all of them?

Reply:
in future, if you have more `prospect-events` that you want to send to this SNS, you can just add them to `detail-type` of this rule in the config file. 

and yes, targetId need to be unique and tied to the `target`. https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Target.html. multiple rules should be able to use the same target

Reply:
i'm also curious about this. guessing the answer is "no", but 🤷 .